reportssql/report_util_functions.sql

```diff
@@ -1625,3 +1625,145 @@ BEGIN
     RETURN result;
 END$$
 DELIMITER ;
+
+-- patientRegisteredBeforeReportStartDate
+
+DROP FUNCTION IF EXISTS patientRegisteredBeforeReportStartDate;
```

Comment:
@tmvumbi2 By enforcing the condition that the child has to be registered before the reporting period start date, aren't we excluding the children registered during the reporting period that have a viral test done during the period? As these children won't be counted for the next quarter.

Reply:
Richard confirmed this. We just check that the test date is during the reporting period, as the child needs to be registered to be able to have a test done.

This function can be removed.